src/contracts/queries.ts

```diff
@@ -59,6 +58,10 @@ export const contractsQueries = {
   }): Sablier.Contract[] | undefined => {
     const { protocol, chainId, release } = opts || {};
 
+    if (chainId && !protocol && !release) {
+      throw new Error("Cannot specify only chainId without protocol or release");
+    }
+
```

Comment:
Instead of throwing an error, we can actually return all contracts for the `chainId`.

Reply:
Sounds good.

Reply:
Then we should add the following comment to it:

```ts
* - { chainId } ⇒ all for that chain ID
```